cardano-cli/test/cardano-cli-golden/files/golden/help/conway_governance_vote_view.cli

```diff
@@ -1,11 +1,13 @@
-Usage: cardano-cli conway governance vote view [--yaml]
+Usage: cardano-cli conway governance vote view [--output-format STRING]
```

Comment:
A flag here would be nicer imo: `--yaml | --json`. We can add flags if we ever want to include more format options.

Reply:
Sounds good, I will do that in a follow-up PR rn, because it's a bigger change, because it affects multiple commands.